docs/config.md

```diff
@@ -207,6 +207,27 @@ The partner registry requires authentication for pulling images.  This can be ac
 docker --config=/var/lib/origin/.docker login -u <registry-user> -p <registry-password> registry.connect.redhat.com
 ```
 
+### API V2 Registry
+
+Registries that implements the Docker [Registry HTTP API V2](https://docs.docker.com/registry/spec/api/) protocol can be configured with the `apiv2` registry adapter as follows
+
+```yaml
+registry:
+  - name: jfrog
+    type: apiv2
+    url:  <registry_url>
+    user: <registry-user>
+    pass: <registry-password>
+    white_list:
+      - ".*-apb$"
+```
+
+If the registry requires authentication for pulling images.  This can be achieved by running the following command on every node in your existing OpenShift cluster:
```

Comment:
s/.  This/, this/

Reply:
Is this OpenShift-specific? Or would this apply to other kubernetes clusters too?